[CCMV] Update CCM _index.md page to include billing conductor (#18870)
DataDog/documentation: content/en/cloud_cost_management/_index.md

[4]: https://docs.aws.amazon.com/cur/latest/userguide/view-cur.html
[5]: https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_create-console.html
[6]: https://docs.aws.amazon.com/cur/latest/userguide/data-dictionary.html

### Billing conductor

> Billing conductor enables you to simplify the bill by customizing the billing rates, distribute credits and fees, and shared overhead costs at your discretion. You will also be able to select which accounts to include in the CUR.

Reviewer: ```suggestion
Billing conductor enables you to simplify your bill by customizing the billing rates, distributing credits and fees, and sharing overhead costs at your discretion. You can also select which accounts to include in the CUR.
```

> To create a billing conductor CUR, please follow [these instructions](https://docs.aws.amazon.com/cur/latest/userguide/cur-data-view.html) from AWS. Make sure the CUR meets [Datadog's requirements](https://docs.datadoghq.com/cloud_cost_management/?tab=aws#prerequisite-generate-a-cost-and-usage-report).
> Once the billing conductor CUR is created, please follow the instructions above to set it up in Datadog.
> 

Reviewer: ```suggestion
To create a billing conductor CUR, follow the [AWS Cost and Usage Reports user guide][1]. Ensure the CUR meets [Datadog's requirements][2].
Once the billing conductor CUR is created, follow the Cloud Cost Management instructions above to set it up in Datadog.

[1]: https://docs.aws.amazon.com/cur/latest/userguide/cur-data-view.html
[2]: https://docs.datadoghq.com/cloud_cost_management/?tab=aws#prerequisite-generate-a-cost-and-usage-report
```

[4]: https://docs.aws.amazon.com/cur/latest/userguide/view-cur.html
[5]: https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_create-console.html
[6]: https://docs.aws.amazon.com/cur/latest/userguide/data-dictionary.html

> ### Billing conductor

Reviewer: ```suggestion
## Billing conductor
```
